src/index.js

```diff
@@ -1 +1,13 @@
 import './style.css';
+import Players from './functions.js';
+
+Players.createNewGame();
+
+const btnRefresh = document.querySelector('#refresh');
+btnRefresh.addEventListener('click', Players.getPlayer);
+
+const btnSubmit = document.querySelector('#btn-submit');
+btnSubmit.addEventListener('click', (e) => {
+  e.preventDefault();
+  Players.addNewPlayer();
+});
```

Comment:
New data is properly being submitted to the API and after the refresh button is clicked, new info is rendered as expected 🙌👌 Fantastic job!!